src/components/kompas-grace-period/kompas-grace-period.tsx

```diff
@@ -79,13 +79,15 @@ export class KompasGracePeriod {
   }
 
   private gracePeriodTemplate = (): void => (
-    <div class="flex flex-col md:flex-row w-full bg-orange-100 justify-end py-4 md:space-x-4 px-4 lg:px-20 bottom-0">
-      <div class="text-grey-600 text-sm md:text-base self-center text-left">{this.getCountdownGracePeriod()}</div>
-      <div class="flex self-center w-full md:w-1/2 justify-end pt-4 md:pt-0">
-        <button onClick={this.redirectToBerlangganan} class="bg-green-500 p-2 px-5 rounded-md font-bold text-grey-100 text-sm md:text-base w-full md:w-auto">
-          Perbarui Langganan
-        </button>
+    <div class="mt-4">
+      <div class="flex flex-col md:flex-row w-full bg-orange-100 rounded-xl justify-center py-4 md:space-x-4 px-4 bottom-0">
+        <div class="text-grey-600 text-sm md:text-base self-center text-left">{this.getCountdownGracePeriod()}</div>
       </div>
+      <div class="self-center w-full pt-4 md:pt-0">
+            <button onClick={this.redirectToBerlangganan} class="w-full rounded-lg px-4 py-3 mt-4 h-10 flex justify-center items-center bg-green-500 text-grey-100 font-bold text-base focus:outline-none">
```

Comment:
```suggestion
    <div class="flex flex-col md:flex-row w-full bg-orange-100 justify-end py-4 md:space-x-4 px-4 lg:px-20 bottom-0">
      <div class="text-grey-600 text-sm md:text-base self-center text-left">{this.getCountdownGracePeriod()}</div>
      <div class="flex self-center w-full md:w-1/2 justify-end pt-4 md:pt-0">
        <button onClick={this.redirectToBerlangganan} class="bg-green-500 p-2 px-5 rounded-md font-bold text-grey-100 text-sm md:text-base w-full md:w-auto">
```

revert to existing style 
![image](https://github.com/pt-kompas-media-nusantara/kompas-web-components/assets/22672170/40726144-e7c5-427a-9305-5883ef1e1e6d)

kalo pake kode baru jadi seperti ini
![image](https://github.com/pt-kompas-media-nusantara/kompas-web-components/assets/22672170/a546c44b-2252-4386-9bdc-0282fd59d155)

tidak sesuai figma squad F terkait grace period : https://www.figma.com/file/qr0KwXmWvCVCt0EWGa6IGL/Grace-Period?type=design&node-id=17-4163&t=HWhxcROSZBIzfSIl-0

cc : @adiandroo @Bayusetiawan45

Reply:
oh sorry mas Andy, yang ditampilin di aku rada beda jadi bulak balik, sorry ya mas